pom.xml

```diff
@@ -410,8 +421,8 @@
 							<locations>
 								${location}
 							</locations>
-							<schemes>http</schemes>
-							<host>localhost:8080</host>
+							<schemes>httpx</schemes>
+							<host>localhostx</host>
```

Comment:
change it to localhost:8080 and add all the required dev enx variables in your local machine